### ISSUE TITLE
[Req]: Portugas.org

### ISSUE THREAD

**Issue by jpf4e** [Jackett/Jackett#10486]
There is a new Portuguese (EU) tracker: Portugas.org
Could you add support for it?

**Comment by garfield69** [comment]
it is sooooo slow, may time out the await response timer.
engine  U-232 V4
getting Error 522 Connection timed out trying to read mail, read rules
MST ?
MR ?
tbd

**Comment by JpF4E** [comment]
Yeah it is rather new... so they're still working on it performance-wise. :)

**Comment by JpF4E** [comment]
It seems a bit better now, if you wish to try it out.

**Comment by Norin-Radd** [comment]
they were under attack. Maybe that was the issue.

**Comment by helsan74** [comment]
They just upgraded to a new server and platform.

**Comment by garfield69** [comment]
new platform is UNIT3D but they have obfuscated the version which is such a pain in the ass. I'll base it off the latest release 5.1.0 and work backwards if I get into trouble.
MR 1.0 MST 3 days

**Comment by garfield69** [comment]
v0.17.598

**Comment by garfield69** [comment]
https://portugas.org/ > cloudflare Web server is down Error code 521

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
online
